server/neural_network_interface.py

```diff
@@ -33,7 +33,7 @@ def generate_move(self, sid, gesture_data, message_data):
 
 class NeuralNetworkBlocksworldModel(BlocksworldModel):
     def __init__(self, h5_paths, ambiguity_threshold=0.3):
-        super().__init__()
+        super(NeuralNetworkBlocksworldModel, self).__init__()
```

Comment:
With Python 3 you don't need to pass the arguments. https://www.python.org/dev/peps/pep-3135/

Reply:
For whatever reason, this was happening when I didn't pass any arguments: https://travis-ci.org/RHIT-XPrize/rhit-xprize-blocksworld/builds/367785839#L782

Reply:
A couple lines above your link I am seeing `/usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py:144:`, is it possible that this is being run with python 2 instead of 3?

Reply:
I think the problem was I was calling `sudo python` at the time (it's a long story), and for whatever reason that stuck it with Python 2.7. It should be fixed now.